docs/linux/zbd-api.md

```diff
@@ -6,24 +6,26 @@ sidebar_label: Zoned Block Device User Interface
 
 # Zoned Block Device User Interface
 
-User applications can access the zone information of a zoned block device and
-can manage the zones of a zoned block device by using two types of interfaces.
+User applications use two types of interfaces to access the zone information of
```

Comment:
s/use/can use

given that applications have the choice...

Reply:
I've chewed on this for a full day, and I see your point.